targets/csharp/source/PlayFabSDK/source/PlayFabUtil.cs.ejs

```diff
@@ -154,5 +155,39 @@ namespace PlayFab
             }
             return Sb.ToString();
         }
+
+        public struct SynchronizationContextRemover : INotifyCompletion
```

Comment:
Should this be a class?

Reply:
This is a copy of known/good MS authored code, from an expert on this particular topic.
I avoided changing anything, such as struct/class.